docs/api.rst

```diff
@@ -4991,6 +4975,106 @@ Dealing with History Events
+- :reqjson list[string] event_identifiers: An optional list of event identifiers to filter for.
+- :reqjson list[string] event_types: An optional list of event types by which to filter the decoded events.
+- :reqjson list[string] event_subtypes: An optional list of event subtypes by which to filter the decoded events.
+- :reqjson list location: An optional location name to filter events only for that location.
+- :reqjson list[string] location_labels: A list of location labels to optionally filter by. Location label is a string field that allows you to provide more information about the location. When used in blockchains, it is used to specify the user's address. For exchange events, it's the exchange name assigned by the user.
+- :reqjson object entry_types: An object with two keys named 'values' and 'behavior'. 'values' is a list of entry types to optionally filter by. 'behavior' is optional and is a string with the value 'include' or 'exclude' which defines the filtering behavior. It defaults to 'include'. Entry type is the event category and defines the schema. Possible values are: "history event," "evm event," "eth withdrawal event," "eth block event," "eth deposit event."
+- :reqjson string asset: The asset to optionally filter by.
+- :reqjson list[string] tx_hashes: An optional list of transaction hashes to filter for. This will make it an EVM event query.
+- :reqjson list[string] counterparties: An optional list of counterparties to filter by. List of strings. This will make it an EVM event query. We currently have a special exception for "eth2" as a counterparty. It filters for all eth staking events if given. It can't be given along with other counterparties in a filter. Or with an entry types filter.
+- :reqjson list[string] products: An optional list of product type to filter by. List of strings. This will make it an EVM event query.
+- :reqjson list[string] addresses: An optional list of EVM addresses to filter by in the set of counterparty addresses. This will make it an EVM event query.
+- :reqjson list[int] validator_indices: An optional list of validator indices to filter by. This makes it an EthStakingevent query.
+
+Exporting History Events
+============================================
+
+.. http:post:: /api/(version)/history/events/export
+
+   Doing a POST on this endpoint with the given filter parameters will export a csv with all history events matching the filter to a file in the provided directory. Only the 'directory_path' argument is required. If no filter is used all the events will be exported.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/history/events/export HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {
+          "directory_path": "/home",
+          "from_timestamp": 1500,
+          "to_timestamp": 999999
+      }
+
+   .. _history_export_schema_section:
```

Comment:
you should also use the link in the other endpoints where you saw them. Not just in the new endpoints.

Reply:
so the set of filters we use here were only used in these two